hil/ext/obm/mock.py

```diff
@@ -50,22 +52,30 @@ def require_legal_bootdev(self, dev):
     def set_bootdev(self, dev):
         return
 
-    @no_dry_run
     def start_console(self):
+        global CONSOLE
+        CONSOLE = True
```

Comment:
The all-caps convention normally indicates something that's a constant, so it shouldn't be used for a variable that you're modifying.

It would also be better for this to operate on a per-obm level, as opposed to having start/stop console toggle *all* obms. Maybe have a look at how the mock switch driver does this (it has a dictionary).